docs/modules/operation/pages/admin/housekeeping/introduction.adoc

```diff
@@ -0,0 +1,43 @@
+
+[[housekeeping]]
+= Housekeeping Tasks
+
+There are a number of housekeeping tasks you may want to do regularly to ensure optimum system performance.
+We also recommend you complete some of these housekeeping tasks before upgrading {page-component-title}.
+
+== Prior to upgrading
+
+[[prune-events]]
+=== Prune unneeded events
+Use vacuumd or cron to regularly prune unneeded events.
+This helps to clean up your events table, and reduces the amount of data in your backups.
+We also recommend doing this before you begin the {page-component-title} upgrade process.
+
+For example, run the following to delete events older than six weeks that have no associated outages:
+
+[source, sql]
+----
+--# this deletes any events that are not associated with outages
+DELETE FROM events WHERE NOT EXISTS
+(SELECT svclosteventid FROM outages WHERE svclosteventid = events.eventid
+UNION
+SELECT svcregainedeventid FROM outages WHERE svcregainedeventid = events.eventid
+UNION
+SELECT eventid FROM notifications WHERE eventid
```

Comment:
need to update SQL example.  This note above mentions 6 weeks, but the sample below doesn't filter by date.  Looks like this line is incomplete.

Reply:
Yup. Missing line.

Reply:
something still looks off.  There is a `(` but no closing parenthesis.

Reply:
Gee, if only there were something like cut and paste and I knew how to use it properly! I think everything is there now.